assets/thanos-ruler/thanos-ruler-prometheus-rule.yaml

```diff
@@ -37,7 +37,7 @@ spec:
         )
       for: 5m
       labels:
-        severity: critical
```

Comment:
jsonnet file lock did not change? 🤔

Reply:
Also no need to bump this, as I will do as part of the release checklist thing.

Reply:
this is what happens when you push code changes during a meeting :smile:

Reply:
haha :D